### PR TITLE
set do not suppress accessibility services for uiautomation

### PR DESCRIPTION
**Pull request by KazuCocoa**
Espresso driver has a couple of methods which call `UiDevice.getInstance`, https://github.com/appium/appium/issues/4910 may happen.

This affects below methods.

- `GetAlertText`/`DismissAlert`/`AcceptAlert`
- `Uiautomator` by mobile command

### CODE REVIEW

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/InteractionHelper.kt [hunk 1]
```diff
@@ -9,6 +12,12 @@ object InteractionHelper {
     @Synchronized
     fun getUiDevice(): UiDevice {
         if (uiDevice == null) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                // The flag is necessary not to stop running accessibility service
+                // https://github.com/appium/appium/issues/4910
+                // https://developer.android.com/reference/android/app/UiAutomation.html#FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES
+                Configurator.getInstance().uiAutomationFlags = UiAutomation.FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES
```
Comment (mykola-mokhnach) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/InteractionHelper.kt]: we probably should do 

```
                Configurator.getInstance().uiAutomationFlags = Configurator.getInstance().uiAutomationFlags | UiAutomation.FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES;
```

In order to not corrupt the current flags

Comment (mykola-mokhnach) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/InteractionHelper.kt]: I would also prefer to have this option configurable to make sure it does not affect current tests

**Comment by KazuCocoa** [comment]
Roughly tried to define a custom runner to handle `-e` when the server launches so that we can handle `UiAutomation.FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES` in launch time, but dex error happened.
https://github.com/appium/appium-espresso-driver/pull/468/commits/da3399f8993269e8141a7200cd21a63f635bf70e

```
2019-11-06 23:58:02.077 20296-20296/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: io.appium.android.apis, PID: 20296
    java.lang.RuntimeException: Unable to instantiate instrumentation ComponentInfo{io.appium.espressoserver.test/io.appium.espressoserver.test.helpers.AppiumAndroidJUnitRunner}: java.lang.ClassNotFoundException: Didn't find class "io.appium.espressoserver.test.helpers.AppiumAndroidJUnitRunner" on path: DexPathList[[zip file "/system/framework/android.test.runner.jar", zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/system/framework/android.test.mock.jar", zip file "/data/app/io.appium.espressoserver.test-SyO1i6Q-0KdJlgOyW9ikag==/base.apk", zip file "/data/app/io.appium.android.apis-_UT2k1A5_pGIrMWDHQvbMQ==/base.apk"],nativeLibraryDirectories=[/system/lib64]]
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5806)
        at android.app.ActivityThread.access$1100(ActivityThread.java:199)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "io.appium.espressoserver.test.helpers.AppiumAndroidJUnitRunner" on path: DexPathList[[zip file "/system/framework/android.test.runner.jar", zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/system/framework/android.test.mock.jar", zip file "/data/app/io.appium.espressoserver.test-SyO1i6Q-0KdJlgOyW9ikag==/base.apk", zip file "/data/app/io.appium.android.apis-_UT2k1A5_pGIrMWDHQvbMQ==/base.apk"],nativeLibraryDirectories=[/system/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5802)
        at android.app.ActivityThread.access$1100(ActivityThread.java:199) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6669) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
```

**Comment by mykola-mokhnach** [comment]
@KazuCocoa It this PR still relevant?

**Comment by KazuCocoa** [comment]
https://github.com/appium/appium-espresso-driver/pull/559

**Pull request closed without merging**